### PR TITLE
Update: RGI_MAIN to ensure you can separate temp files from real output 

### PR DESCRIPTION
**Pull request by jfy133**
As `publishDir`  patterns can't exclude (or rather globs don't :().

Moves all files with `temp` in the name to a dedicated directory

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/rgi/main/main.nf [hunk 1]
```diff
@@ -13,6 +13,7 @@ process RGI_MAIN {
     output:
     tuple val(meta), path("*.json"), emit: json
     tuple val(meta), path("*.txt") , emit: tsv
+    tuple val(meta), path("temp/")  , emit: tmp
```
Comment (jasmezz) [inline modules/nf-core/rgi/main/main.nf:16]:
```suggestion
    tuple val(meta), path("temp/") , emit: tmp
```

**Review by jasmezz — APPROVED** [review]
LGTM

**Comment by jasmezz** [comment]
Any idea why conda test is failing? (Logs: `Failed to create Conda environment`)

**Comment by jfy133** [comment]
Runnig out of memory with `137`... bizareee.. works locally for me on conda though, so i will just merge!

**Pull request merged**